list2.py

```diff
@@ -18,8 +18,16 @@
 
 
 def remove_adjacent(nums):
-    """Your code goes here.  Edit this docstring."""
-    return
+    list = []
```

Comment:
using `list` as a variable is not good for python -- you are changing default behavior ..  pick variable names that do not collide with Python's reserved keywords, such as `list_`

Reply:
I appreciate it 💯 
I'll use that.